### PR TITLE
Turn ziektenplagenexotengroen classes into layers

### PR DESCRIPTION
**Pull request by lbam**
Also, generate this file. It's too repetitive to type and mapfile syntax isn't powerful enough (AFAICT) to parametrize an included file.

For [AB#77375](https://dev.azure.com/CloudCompetenceCenter/089b38ee-0066-4b66-a36c-20744a0e4348/_workitems/edit/77375). Closes #352.

### CODE REVIEW

**Comment by lbam** [comment]
image

File: gen_ziektenplagenexotengroen.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    ("Eikenprocessierups aanwezig (Standaard)", "caterpillar_orange"),
+]
+
+print("# GENERATED FILE, DO NOT EDIT.\n\n")
+print("# TEAM: Bor / Beeldschoon\n")
+
+with block("MAP"):
+    p("NAME", "ziektenplagenexotengroen")
+    p("INCLUDE", "header.inc")
+    p("DEBUG", 5)
```
Comment (jjmurre) [inline gen_ziektenplagenexotengroen.py:56]: Does this need to stay here for the final version?

**Comment by jjmurre** [comment]
Nice! Much more DRY.

File: gen_ziektenplagenexotengroen.py (hunk 1 above)
Comment (lbam) [reply gen_ziektenplagenexotengroen.py:56]: There's lots of different `DEBUG` settings in this repo already. I intend to clean all of them up at a later point.

**Review by jjmurre — APPROVED** [review]

**Pull request merged**